packages/core/src/indexing-store/realtime.ts

```diff
@@ -137,8 +137,7 @@ export const createRealtimeIndexingStore = ({
                         .onConflictDoNothing()
                         .returning();
                     } catch (e) {
-                      parseSqlError(e);
-                      throw e;
+                      throw parseSqlError(e);
```

Comment:
Noticed that we were double-throwing here (parseSqlError was throwing internally). I think it makes more sense to return the error from the helper function and then throw it?